### DISCUSSION TITLE
exit code (0xC0000005) when calling UnstructuredGrid.plot()

### DISCUSSION THREAD

**Discussion by tboussaid** [pyvista/pyvista#2401] [Q&A] [answered]
Hi, 

I am trying to plot a mesh from a cgns file. The mesh elements have mixed connectivity. After doing the appropriate cgns to vtk mapping, I create the unstructured grid as follows : 
``` 
pyvista_mesh = pyvista.UnstructuredGrid(
                    np.hstack(faces), celltypes, vertices
                ) 
```
When I try to plot it, I get this exit code message in Pycharm 
``` 
Process finished with exit code -1073741819 (0xC0000005)
``` 
Any ideas how to solve this problem ? 

Thank you in advance for your help !

**Comment by akaszynski** [comment, marked as answer]
Without seeing the data, I'm guessing that there's an issue with the integrity of the data. Check to make sure that your face indices are all valid.

**Reply by benjika** [reply]
can you please explain what is data integrity?
in my case, I take a surface and make sure all normals go up, then seal it from below with a plane. when i thy to do boolean action it has this error

**Comment by tboussaid** [comment]
Hi @akaszynski, 
Thanks for your reply. 
It is an issue with the data integrity in fact. I have found some quad_4 faces with only 3 nodes.
